burn-dataset/Cargo.toml

```diff
@@ -18,7 +18,7 @@ audio = ["hound"]
 fake = ["dep:fake"]
 
 sqlite = ["__sqlite-shared", "dep:rusqlite"]
-sqlite-bundled = ["__sqlite-shared", "rusqlite/bundled"]
+sqlite-bundled = ["__sqlite-shared", "rusqlite?/bundled"]
```

Comment:
I think the `?` is making the tests fail, and it's a bit unnecessary in this case.

Reply:
Thanks. Not sure if my Rust is too new.

According to the [documentation] (https://doc.rust-lang.org/cargo/reference/features.html), 
```
You can add a ? as in "package-name?/feature-name" which will only enable the given feature if something else enables the optional dependency.
```

Reply:
No Rust isn't too new, it's just that `rusqlite` is necessary for the feature flag to work I think.